### ISSUE TITLE
Decrease citizen fertility

### ISSUE THREAD

**Issue by dymanoid** [dymanoid/RealTime#138]
Currently, an young/adult pair can make babies pretty often. This can be even multiple times a day! Not so real...

The citizens fertility needs to be controlled. But not too aggressive; otherwise, it might take too long for a city to evolve.

**Comment by dymanoid** [comment]
The reason why you see that birth rate drop is the fact that the citizens aging has changed. Now the citizens grow-up slowly, so you have less adults but significantly more children. But as soon as your city has lots of adults again (after approx. 20 game days), the birth rate will become unrealistically high either.

I don't know how you calculate the "life span" and what does it actually mean. In **Real Time**, each game day equals to one citizen's year, regardless of the time speed you select. However, if you use the Lifecylce Rebalance mod, **Real Time** disables its control over citizens age and passes the control to that mod. But note that Lifecycle Rebalance has no idea of the time speed selected in **Real Time**, so at speed 1 citizens can live e.g. 2 days long (just a random number) while at speed 6 they will live 64 days long. Each time speed level is 2 times faster that previous one.

This feature is already implemented and will be released with the next version.

**Comment by benzoll37** [comment]
If 20 days into the game increases fertility, then I did not say anything, hehe.

I do not use "Rebalance" with "RT". Lately I used "Lifespan multiplier v1.1" + "Time Warp Fix", but now I do not use it with "RT".
Life x1 or x3 or x36 is a reference that we all know by the mods (configurable) that there is about increase in life span.

My calculations were made with mod "Favorite Cims": each citizen allows an exhaustive follow-up, including his age.
-Vanilla game x1 = cims life x time (real, not in game)
- "Lifespan multiplier v1.1" x3 = cims last three times longer (real, not in game)
- "Real Time": cims last about 12 times longer (real, not in play).
Maybe this calculation is not valid for RT.
